### PR TITLE
[NIH.gov] Update and reactivate ruleset

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

**Comment by koops76** [comment]
#11114

**Comment by Bisaloo** [comment]
I am pretty sure nearly all subdomains work over `https` now but I am not sure I want to invest the time to check and add them all given that it is likely that this domain will be preloaded soon (as all others `.gov` rulesets).

File: src/chrome/content/rules/National-Institutes-of-Health.xml [hunk 1]
```diff
@@ -222,43 +161,62 @@ Fetch error: http://local-rximage.nlm.nih.gov/ => https://local-rximage.nlm.nih.
...
 
-	<!--	Direct rewrites:
-				-->
+	<target host="nih.gov" />
+	<target host="www.nih.gov" />
 	<target host="aidsinfo.nih.gov" />
 	<target host="calendar.nih.gov" />
+	<target host="cancergenome.nih.gov" />
 	<target host="commonfund.nih.gov" />
 	<!--target host="internet.csr.nih.gov" /-->
```
Comment (J0WI) [inline src/chrome/content/rules/National-Institutes-of-Health.xml]: Obsolete comment

File: src/chrome/content/rules/National-Institutes-of-Health.xml [hunk 2]
```diff
@@ -222,43 +161,62 @@ Fetch error: http://local-rximage.nlm.nih.gov/ => https://local-rximage.nlm.nih.
...
-	<target host="intrahr.nih.gov" />
 	<target host="irp.nih.gov" />
 	<target host="itrusteauth.nih.gov" />
+
 	<target host="jobs.nih.gov" />
 	<target host="www.jobs.nih.gov" />
+
 	<target host="list.nih.gov" />
 	<target host="www.lrp.nih.gov" />
-	<target host="temp.my.nih.gov" />
```
Comment (J0WI) [inline src/chrome/content/rules/National-Institutes-of-Health.xml]: Could you add a comment for it?

File: src/chrome/content/rules/National-Institutes-of-Health.xml [hunk 3]
```diff
@@ -268,41 +226,47 @@ Fetch error: http://local-rximage.nlm.nih.gov/ => https://local-rximage.nlm.nih.
 	<target host="nei.nih.gov" />
 
 	<target host="answers.nei.nih.gov" />
-	<target host="nationaleyegene.nei.nih.gov" />
 	<target host="www.nei.nih.gov" />
 
 	<target host="newsinhealth.nih.gov" />
-	<target host="catalog.nhlbi.nih.gov" />
```
Comment (J0WI) [inline src/chrome/content/rules/National-Institutes-of-Health.xml]: Please keep a comment for it

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**